oprah-proxy.py

```diff
@@ -2,19 +2,18 @@
 
 import base64
 import hashlib
-import urllib.parse
 import uuid
+import logging
 
 try:
     import requests
 except ImportError:
-    print('ERROR: Cannot import requests')
-    print('DEBUG: Please install Requests for Python 3, see '
-          'http://docs.python-requests.org/en/master/user/install/ or use your '
-          'favorite package manager (e.g. apt-get install python3-requests)')
+    logging.error('Cannot import requests')
+    logging.debug('Please install Requests for Python 3, see '
+                  'http://docs.python-requests.org/en/master/user/install/ or use your '
+                  'favorite package manager (e.g. apt-get install python3-requests)')
     exit(2)
 
-
```

Comment:
Looks like [PEP 8 requires 2 blank lines](https://www.python.org/dev/peps/pep-0008/#blank-lines) here, can you put it back?